logconfig.py

```diff
@@ -65,19 +65,18 @@
         # Use this to set different log levels without changing 'real' code.
         "myclasses": {
             "level": "DEBUG",
-            "propagate": "no"
+            "propagate": False
```

Comment:
Ah, it turns out that what I really want here is "propagate": True. As you can see from the example I create a logger called myclasses.MyClass  - and in order for that one to work I need the propagation switched on. So it just happened to work before because "no" evaluates to True...